hs_swat_modelinstance/models.py

```diff
@@ -46,12 +46,18 @@ def get_swat_model_objectives(self):
 
     @classmethod
     def _add_swat_objective(cls, model_objective, objectives):
```

Comment:
Looking at the code it seems this method is being called both in ```create()``` and ```update()``` functions of the ```ModelObjective``` class. I am wondering how the validation is successful in the function ```_validate_swat_model_objectives()``` when the list of values passed into this function are of type integers.